packages/block-library/src/paragraph/edit.native.js

```diff
@@ -7,7 +7,6 @@ import {
 	AlignmentToolbar,
```

Comment:
The new hook is implemented in several blocks. I guess we might need similar removals in other blocks (heading, group, columns, media & text)

Reply:
It depends on how the block was implemented, for example heading now shares the same edit code as the web, so no changes needed.